packages/syft/src/syft/service/worker/worker_image_service.py

```diff
@@ -155,3 +219,81 @@ def delete(
         )
 
         return SyftSuccess(message=returned_message)
+
+    def docker_build(
+        self, image: SyftWorkerImage
+    ) -> Union[Tuple[Image, Iterable[str]], SyftError]:
+        if not isinstance(image.config, DockerWorkerConfig):
+            # Handle this to worker with CustomWorkerConfig later
+            return SyftError("We only support DockerWorkerConfig")
```

Comment:
Since `docker_build` is now integrated with `CustomImageBuilder`, we can build any configuration - cog-like and dockerfile. let me know if we want to remove this.